SU2_CFD/src/solvers/CTransLMSolver.cpp

```diff
@@ -314,8 +336,13 @@ void CTransLMSolver::Source_Residual(CGeometry *geometry, CSolver **solver_conta
 
     numerics->SetStrainMag(flowNodes->GetStrainMag(iPoint), 0.0);
 
-    /*--- Set coordnate (for debugging) ---*/
+    /*--- Set coordinate (for debugging) ---*/
     numerics->SetCoord(geometry->nodes->GetCoord(iPoint), nullptr);
+
+    if(options.LM2015){
+      /*--- Set local grid length (for LM2015)*/
+      numerics->SetLocalGridLength(geometry->nodes->GetMaxLength(iPoint), 0.0);
```

Comment:
If this only needs one value please change SetLocalGridLength to take one value instead of two.